main/src/ca/uwaterloo/flix/language/errors/ResolutionError.scala

```diff
@@ -311,6 +311,27 @@ object ResolutionError {
     }
   }
 
+  /**
+    * Undefined Class Error.
+    *
+    * @param clazz  the class.
+    * @param sig    the unresolved sig.
+    * @param ns     the current namespace.
+    * @param loc    the location where the error occurred.
+    */
+  case class UndefinedSig(clazz: Name.QName, sig: Name.Ident, ns: Name.NName, loc: SourceLocation) extends ResolutionError {
+    def summary: String = "Undefined sig."
+    def message: VirtualTerminal = {
+      val vt = new VirtualTerminal
+      vt << Line(kind, source.format) << NewLine
+      vt << ">> Undefined sig '" << Red(clazz.toString + "." + sig.name) << "'." << NewLine
```

Comment:
How about 

Undefined signature 'sig' in class 'class'.

Reply:
I guess our use of the word signature is also not completely correct. But let us worry about that later.